Dockerfile

```diff
@@ -1,28 +1,32 @@
-FROM node:8
+FROM golang:alpine as golang
+RUN apk add --no-cache build-base git && \
+    go get github.com/tj/node-prune/cmd/node-prune
 
-ENV HOST localhost
-ENV PORT 3000
-
-# Create app directory
-RUN mkdir -p /usr/src/app
+FROM node:8-alpine as build
 WORKDIR /usr/src/app
-
-# Install GYP dependencies globally, will be used to code build other dependencies
-RUN npm install -g --production node-gyp && \
```

Comment:
Needs `-ex` instead. Otherwise if any of these commands fail, the docker image continues building.

Reply:
Good point, I'm actually not too familiar with npm, my core strength lays with php & infrastructure. I was actually surprised things just continued even if there was an error.

Reply:
Sorry! I was mistaken, you are using `&&`, I generally use `;` to separate them. This confused me. They should behave the same way though :)